### PR TITLE
docs: add headless as deprecated

### PR DESCRIPTION
**Pull request by santoshyadavdev**

### CODE REVIEW

**Comment by jordanpowell88** [comment]
@santoshyadavdev It looks like the CSS/Markdown step is failing. Can you run `yarn lint` and sign our `CLA`?

**Comment by santoshyadavdev** [comment]
> @santoshyadavdev It looks like the CSS/Markdown step is failing. Can you run `yarn lint` and sign our `CLA`?

Done

File: content/guides/guides/command-line.md [hunk 1]
```diff
@@ -116,7 +116,7 @@ cypress run [options]
 | `--env`, `-e`              | [Specify environment variables](#cypress-run-env-lt-env-gt)                                                                                                                                |
 | `--group`                  | [Group recorded tests together under a single run](#cypress-run-group-lt-name-gt)                                                                                                          |
 | `--headed`                 | [Displays the browser instead of running headlessly](#cypress-run-headed)                                                                                                                  |
-| `--headless`               | [Hide the browser instead of running headed (default during `cypress run`)](#cypress-run-headless)                                                                                         |
+| `--headless`               | [Hide the browser instead of running headed (default during `cypress run`)](#cypress-run-headless) this option is deprecated                                                               |
```
Comment (emilyrohrbough) [inline content/guides/guides/command-line.md:119]: @santoshyadavdev I just double check our CLI options & this option isn't deprecated. Did you come across something that indicated this?

Comment (santoshyadavdev) [reply content/guides/guides/command-line.md:119]: Yes actually saw the message from CLI that this option is deprecated.

**Pull request closed without merging**